docs/tutorial/01-story_graph.rst

```diff
@@ -0,0 +1,130 @@
+.. _Story Graph:
+
+The story graph
+===============
+
+The story graph acts as a central element of Gencaster to create and control the non-linear storytelling experiences of Gencaster.
+It creates and negates possibilities through connections and statements within this graph.
```

Comment:
negates or negotiates?

Reply:
negates - making everything possible may seem desirable, but actually good software omits possibilities in order to express a certain idea more easily :)